recipes/opensim/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "3.3" %}
+
+package:
+    name: opensim
+    version: {{ version }}
+
+source:
+    fn: OpenSim-{{ version }}-source.zip
+    url: https://simtk.org/frs/download_start.php/file/5025/OpenSim-{{ version }}-source.zip
+    sha256: 597482e78202f8480d205f10084ca4663236d76aa59de00a0408cdefbac43e7c
+
+build:
+    number: 200
+    # NOTE : For C++11 builds, vc14/VS 2015 are the only acceptable targets
+    # which only Python 3.5 supports. So skip previous Python versions.
+    skip: true                # [win and not py35]
+    features:
+        - vc14                # [win and py35]
+    # NOTE : This is needed to ensure the paths to Simbody's binaries, like
+    # simbody-visualizer, are corrected.
+    detect_binary_files_with_prefix: true
+
+# TODO: Doxygen is the only dependency that is omitted here, so the rendered
+# documentation is not built.
+requirements:
+    build:
+        - toolchain
+        - cmake >=2.8.6
+        - simbody 3.5.4
+        - python                  # [win]
+    run:
+        - simbody 3.5.4
+test:
+  requires:
+    - python 3.5.*  # [win and py35]
+  commands:
+    - test -f $PREFIX/bin/analyze                                             # [unix]
+    - test -f $PREFIX/bin/forward                                             # [unix]
+    - test -f $PREFIX/bin/scale                                               # [unix]
+    - test -f $PREFIX/bin/ik                                                  # [unix]
+    - test -f $PREFIX/bin/id                                                  # [unix]
+    - test -f $PREFIX/bin/cmc                                                 # [unix]
+    - test -f $PREFIX/bin/rra                                                 # [unix]
+    - test -f $PREFIX/lib/libosimLepton.dylib                                 # [osx]
+    - test -f $PREFIX/lib/libosimCommon.dylib                                 # [osx]
+    - test -f $PREFIX/lib/libosimSimulation.dylib                             # [osx]
+    - test -f $PREFIX/lib/libosimActuators.dylib                              # [osx]
+    - test -f $PREFIX/lib/libosimAnalyses.dylib                               # [osx]
+    - test -f $PREFIX/lib/libosimTools.dylib                                  # [osx]
+    - test -f $PREFIX/lib/libosimLepton.so                                    # [linux]
+    - test -f $PREFIX/lib/libosimCommon.so                                    # [linux]
+    - test -f $PREFIX/lib/libosimSimulation.so                                # [linux]
+    - test -f $PREFIX/lib/libosimActuators.so                                 # [linux]
+    - test -f $PREFIX/lib/libosimAnalyses.so                                  # [linux]
+    - test -f $PREFIX/lib/libosimTools.so                                     # [linux]
+    - test -f $PREFIX/sdk/include/Vendors/lepton/include/Lepton.h             # [unix]
```

Comment:
I agree. But this is not likely to change for 3.3. It makes sense from the perspective of an end-user application but not from the perspective of a software library.